beetsplug/discogs.py

```diff
@@ -222,8 +243,11 @@ def get_master_year(self, master_id):
         """
         self._log.debug(u'Searching for master release {0}', master_id)
         result = Master(self.discogs_client, {'id': master_id})
+
+        self.wait_for_rate_limiter()
```

Comment:
Maybe a more semantic name like `request_start` would help explain why the call is here?